static/app/views/replays/replayTable/index.tsx

```diff
@@ -12,15 +12,7 @@ import useOrganization from 'sentry/utils/useOrganization';
 import {useRoutes} from 'sentry/utils/useRoutes';
 import type {ReplayListRecordWithTx} from 'sentry/views/performance/transactionSummary/transactionReplays/useReplaysFromTransaction';
 import HeaderCell from 'sentry/views/replays/replayTable/headerCell';
-import {
-  ActivityCell,
-  DurationCell,
-  ErrorCountCell,
-  ProjectCell,
-  SessionCell,
-  StartedAtCell,
-  TransactionCell,
-} from 'sentry/views/replays/replayTable/tableCell';
+import renderReplayCell from 'sentry/views/replays/replayTable/tableCell';
```

Comment:
The default export is named TableCell, can you make file == default export name == default import name

Reply:
`tableCell.tsx` and `TableCell` all the things 👍